app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def show_user; end
+
+  def show_user_posts; end
+
+  def show_user_post; end
```

Comment:
- [ ] For the best practice create post controller methods in another file.

```ruby
# app/controllers/posts_controller.rb
class PostsController < ApplicationController
  
  def index; end

  def show; end

end
```

Reply:
Fantastic :100:
I am going to do that right away!